### DISCUSSION TITLE
Chia plot not going past %20

### DISCUSSION THREAD

**Discussion by FrozenOwl1** [Chia-Network/chia-blockchain#8485] [Support]
\I started mining chia and my plot won't go past %20 and on computation table 4. my final destination drive has been stuck at 100% for the last day and a half.

**Comment by jack60612** [comment]
> \I started mining chia and my plot won't go past %20 and on computation table 4. my final destination drive has been stuck at 100% for the last day and a half.

100% what?

**Comment by FrozenOwl1** [comment]
its been stuck at 100% usage but nothing has gotten done (no its not full there is still plenty of space)

**Comment by FrozenOwl1** [comment]
is it possible that this stage needs a lot of ram because if it is that might be my issue

**Comment by hoffmang9** [comment]
This is almost always one of a handful of hardware issues. It is usually that your RAM is faulty in one of two ways. It either has bad BIOS settings or you have a faulty RAM stick.

**Comment by FrozenOwl1** [comment]
did a little more digging and I think it's just I don't have enough ram because it's moving at 1 bucket a day so I am gonna try some more ram and see about XMP and maybe a little overclock.

**Comment by loppefaaret** [comment]
ohhh, yes, that makes sense, you are most likely swapping, and that is why your drive is going 100%, but nothing is happening.
either as you suggested add additional memory, or turn down the amount of memory allocated to the plotter - are you on a 4GB system ? 
https://www.chia.net/2021/02/22/plotting-basics.html
this is a writeup that might be worth a read, it also has memory needs for plotting k32 listed.

**Comment by FrozenOwl1** [comment]
Yes I am on a single 4Gb stick so I will try that

**Comment by jack60612** [comment]
> Yes I am on a single 4Gb stick so I will try that

im not sure 4gb will work at all to be honest

**Comment by FrozenOwl1** [comment]
> im not sure 4gb will work at all to be honest

I didn't think it would work either but it is the maximum you can do for a plot

Also i restarted the plot allocated max ram and allocated two more threads and now in a couple hours i have done 6 buckets instead of one a day.

**Reply by loppefaaret** [reply]
> > im not sure 4gb will work at all to be honest
> 
> I didn't think it would work either but it is the maximum you can do for a plot
> 
> Also i restarted the plot allocated max ram and allocated two more threads and now in a couple hours i have done 6 buckets instead of one a day.

I would recommend that you isntead of turning memory usage up, you turn it down, all the way down
go with 900, and a max of 4 threads, and try and limit what other programs is opened during the plotting process, to give the plotter all the memory it might need, when it needs it!

**Reply by FrozenOwl1** [reply]
Thank you loppefaaret that has worked but now I am stuck at 55% and giving me in the log

Only wrote 0 of 1048570 bytes to (temporary drive) with length 328204288. Error 1.

**Reply by FrozenOwl1** [reply]
and turns out its my internet

**Comment by loppefaaret** [comment]
this is turning into a support issue, and not a code issue
therefor im closing th issue, and transferring it to our support discussions instead

**Comment by BrandtH22** [comment]
@FrozenOwl1 , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
